WhisperAPI/WhisperAPI/Controllers/ContextController.cs

```diff
@@ -8,9 +8,9 @@ namespace WhisperAPI.Controllers
 {
     public class ContextController : Controller
     {
-        private Contexts _contexts;
+        private IContexts _contexts;
 
-        public ContextController(Contexts contexts)
+        public ContextController(IContexts contexts)
```

Comment:
How is this injected ? Where is it specified that the InMemoryContext should be injected ?